.github/workflows/ios.yaml

```diff
@@ -90,10 +92,10 @@ jobs:
       - name: "Install dependencies"
         run: ./ci/mac_ci_setup.sh
       - name: Run iOS tests (tsan)
-        run: env -u ANDROID_NDK_HOME ./bazelw test $(./bazelw query 'kind(ios_unit_test, //test/platform/swift/unit_integration/core/...)') --test_tag_filters=macos_only --test_output=errors --config ci --config ios-tsan
+        run: env -u ANDROID_NDK_HOME ./bazelw test $(./bazelw query 'kind(ios_unit_test, //test/platform/swift/unit_integration/core/...)') --test_tag_filters=macos_only --test_output=errors --config ci
   verify_ios:
     runs-on: ubuntu-latest
-    needs: ["macos_tsan", "swift_hello_world"]
+    needs: ["swift_hello_world"]
```

Comment:
this step should still require the job `macos_tsan` that now just runs the tests (like I mentioned we should probably rename the `macos_tsan` step)